src/UI/Component/Glyph/Factory.php

```diff
@@ -0,0 +1,482 @@
+	 *       2: >
+	 *          If clicking on the Back/Next GLYPH opens a previous view of an object, the Back Glyph MUST be used.
+	 *   accessibility:
+	 *       1: >
+	 *          The aria-label MUST be ‘Back'.
+	 * ---
+	 * @param	string|null	$action
+	 * @return 	\ILIAS\UI\Component\Glyph\Glyph
+	 */
+	public function back($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Next Glyph indicates a possible change of the view. The view change leads back to some previous view.
+	 *   composition: >
+	 *       The chevron-right glyphicon is used.
+	 *   effect: >
+	 *       The click on a Next Glyph opens a new view.
+	 *
+	 *
+	 * rules:
+	 *   usage:
+	 *       1: >
+	 *          Back and Next Buttons MUST be accompanied by the respective Back/Next Glyph.
+	 *   style:
+	 *       1: >
+	 *          If clicking on the Back/Next GLYPH opens a new view of an object, the Next Glyph MUST be used.
+	 *       2: >
+	 *          If clicking on the Back/Next GLYPH opens a previous view of an object, the Back Glyph MUST be used.
+	 *   accessibility:
+	 *       1: >
+	 *          The aria-label MUST be ‘Next'.
+	 * ---
+	 * @param	string|null	$action
+	 * @return 	\ILIAS\UI\Component\Glyph\Glyph
+	 */
+	public function next($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Sorting Glyphs indicate the sorting direction of a column in a table as ascending (up) or descending (down).
+	 *       It is a toggle reversing the ordering of a column.
+	 *   composition: >
+	 *       The Sort Ascending Glyph uses glyphicon-arrow-up.
+	 *   effect: >
+	 *       Clicking the Sort Ascending Glyph reverses the direction of ordering in a table.
+	 *
+	 * rules:
+	 *   accessibility:
+	 *       1: >
+	 *          The aria-label MUST be ‘Sort Ascending'.
+	 * ---
+	 * @param	string|null	$action
+	 * @return 	\ILIAS\UI\Component\Glyph\Glyph
+	 */
+	public function sortAscending($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The Sorting Glyphs indicate the sorting direction of a column in a table as ascending (up) or descending (down).
+	 *       It is a toggle reversing the ordering of a column.
+	 *   composition: >
+	 *       The Sort Descending Glyph uses glyphicon-arrow-descending.
+	 *   effect: >
+	 *       Clicking the Sort Descending Glyph reverses the direction of ordering in a table.
+	 *
+	 * rules:
+	 *   accessibility:
+	 *       1: >
+	 *          The aria-label MUST be ‘Sort Descending'.
+	 * ---
+	 * @param	string|null	$action
+	 * @return 	\ILIAS\UI\Component\Glyph\Glyph
+	 */
+	public function sortDescending($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       The User Glyph triggers the “Who is online?” Popover in the Top Navigation.
+	 *       The User Glyph indicates the number of pending contact requests and users online via the the Novelty Counter and Status Counter respectively.
+	 *   composition: >
```

Comment:
When to add a rule and when not is an edge case from time to time. We add a rule, whenever we feel, that there is potential for misuse. We should not add rules for every single aspect in the description otherwise we spam every entry with so many rules, that the important ones (those tackling critical parts) disappear.

Further, as with the attributes of components, I think it is better to start with a minimal set of rules and add new ones if necessary, not the other way around.

In this case however 2 developers already stumbled upon this. The entry contains only one rule so far. Since the usage leaves questions, a rule might be good to clarify things better.

@klees : You are welcome to propose a new rule.